pypesto/objective/priors.py

```diff
@@ -0,0 +1,345 @@
+import numpy as np
+from typing import Callable, Dict, List, Sequence, Tuple, Union
+from copy import deepcopy
+
+from .function import ObjectiveBase
+from .aggregated import AggregatedObjective
+from .constants import MODE_FUN, MODE_RES, FVAL, GRAD, HESS
+
+ResultDict = Dict[str, Union[float, np.ndarray, Dict]]
```

Comment:
I believe this should be importable from somewhere else.

Reply:
it should be defined in the objective module.